packages/frontend/vite.config.ts

```diff
@@ -46,8 +46,6 @@ export default defineConfig({
                     vega: ['vega', 'vega-lite'],
                     ace: ['ace-builds', 'react-ace/lib'],
                     modules: [
-                        'moment/moment.js',
```

Comment:
should `dayjs` be added here? 🤔

Reply:
No need, I think day.js package is tree-shakeable.

Reply:
It's not tree shakeable but it's the usage of a core package that is lightweight + plugins that extend functionality https://github.com/iamkun/dayjs/issues/1235#issuecomment-731605523

Reply:
kool!